clusterloader2/testing/density/config.yaml

```diff
@@ -1,11 +1,19 @@
+# ASSUMPTIONS:
+# - Underlying cluster should have 100+ nodes.
+# - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
+
 #Constants
 {{$NODES_PER_NAMESPACE := 100}}
-{{$PODS_PER_NODE := 30}}
+{{$PODS_PER_NAMESPACE := 3000}}
+{{$MIN_LATENCY_PODS := 500}}
+{{$MIN_SATURATION_PODS_TIMEOUT := 180}}
```

Comment:
I wouldn't call that min. because we add it every time.
OTOH, I don't have suggestion for better name...